pkg/backend/httpstate/stack.go

```diff
@@ -76,6 +77,8 @@ type cloudStack struct {
 	cloudURL string
 	// orgName is the organization that owns this stack.
 	orgName string
+	// currentOperation contains kind, author, and start time of current operation.
```

Comment:
Nit: I'd suggest adding specifics like " contains kind, author, and start time of current operation" since that's the type of comment that will easily become out of date or incorrect as the code evolves. Instead, what is the general purpose of the field. (And the reader will need to drill into the type's definition to get the specifics.)

So I'd suggest instead "contains information about any current operation being performed on the stack, as applicable."